<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gonads</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Gonad Collection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_take_nad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Take Gonad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_nad_wt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gonad Weight</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_done">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Current Barcode</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_cur_code">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
